state-chain/pallets/cf-broadcast/src/lib.rs

```diff
@@ -746,6 +746,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 
 	fn start_next_broadcast_attempt(broadcast_attempt: BroadcastAttempt<T, I>) {
 		let broadcast_id = broadcast_attempt.broadcast_attempt_id.broadcast_id;
+		Self::take_awaiting_broadcast(broadcast_attempt.broadcast_attempt_id);
```

Comment:
This is not required, in both places we call start_next_broadcast_attempt, we already take